index.js

```diff
@@ -59,7 +65,13 @@ module.exports = (input, opts) => {
 				task: () => del('node_modules')
 			},
 			{
-				title: 'Installing dependencies',
+				title: 'Installing dependencies using Yarn',
+				enabled: () => opts.yarn === true,
+				task: () => exec('yarn', ['install'])
```

Comment:
Late to the party, but I'm wondering if it makes sense to add the `--pure-lockfile` flag since rimraf'ing the `node_modules` folder and doing a `yarn install` can generate a different lockfile and will fail the  "Bumping version" step because the Git working directory is not clean.